docs/manual/getting_started.rst

```diff
@@ -1572,11 +1902,11 @@ applications using the C++ binding for Cyclone DDS:
     cmake --build .
 
 The ``<cyclonedds-cxx-install-location>`` is where the C++ binding for
-Cyclone DDS is installed to.
+Cyclone DDS is installed.
 
 To install the package after a successful build:
 
-Depending on the installation location you may need administrator
+Depending on the installation location, you may need an administrator
```

Comment:
I will add an explanation since this has reoccured. The word `privileges` is plural, and administrator is a qualifier of privileges. Since `an` is used with singular nouns, and `administrator privileges` is plural, you should *not* preface it with `an`.